editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -349,17 +431,20 @@ impl Fsm for PathToolFsmState {
 	}
 
 	fn update_hints(&self, responses: &mut VecDeque<Message>) {
+		let general_hint_data = HintData(vec![
+			HintGroup(vec![HintInfo::mouse(MouseMotion::Lmb, "Select Point"), HintInfo::keys([Key::Shift], "Extend Selection").prepend_plus()]),
+			HintGroup(vec![HintInfo::mouse(MouseMotion::LmbDrag, "Drag Selected")]),
+			HintGroup(vec![HintInfo::arrow_keys("Nudge Selected"), HintInfo::keys([Key::Shift], "10x").prepend_plus()]),
+			HintGroup(vec![HintInfo::keys([Key::KeyG, Key::KeyR, Key::KeyS], "Grab/Rotate/Scale Selected")]),
+		]);
+
 		let hint_data = match self {
-			PathToolFsmState::Ready => HintData(vec![
-				HintGroup(vec![HintInfo::mouse(MouseMotion::Lmb, "Select Point"), HintInfo::keys([Key::Shift], "Extend Selection").prepend_plus()]),
-				HintGroup(vec![HintInfo::mouse(MouseMotion::LmbDrag, "Drag Selected")]),
-				HintGroup(vec![HintInfo::arrow_keys("Nudge Selected"), HintInfo::keys([Key::Shift], "10x").prepend_plus()]),
-				HintGroup(vec![HintInfo::keys([Key::KeyG, Key::KeyR, Key::KeyS], "Grab/Rotate/Scale Selected")]),
-			]),
+			PathToolFsmState::Ready => general_hint_data,
 			PathToolFsmState::Dragging => HintData(vec![HintGroup(vec![
 				HintInfo::keys([Key::Alt], "Split/Align Handles (Toggle)"),
 				HintInfo::keys([Key::Shift], "Share Lengths of Aligned Handles"),
 			])]),
+			PathToolFsmState::DrawingBox => general_hint_data,
```

Comment:
These hints are not relevant whilst drawing a box. When drawing a box, there should just be a hint for add to selection.

Reply:
I have added the hints for the `shift` key as well as the `lmb` button, as they are shown in the `SelectTool`. Please let me know if only and only `shift` key's hint is to be shown.